evergreen.yml

```diff
@@ -225,10 +223,20 @@ functions:
       params:
         silent: true
         working_dir: mongo-powerbi-connector
+        env:
+          SELF_SIGNED_PWD: "${self_signed_pwd}"
+          SELF_SIGNED_CERT: "${self_signed_cert}"
+          SELF_SIGNED_KEY: "${self_signed_key}"
+          SELF_SIGNED_PFX: "${self_signed_pfx}"
```

Comment:
```suggestion
```
This was from the troubleshooting I recall, the pfx is generated now that things are working.